docs/client.rst

```diff
@@ -23,3 +23,15 @@ package. Source code for such OpenWrt package can be `found here`_.
 .. _OpenWrt: https://openwrt.org/
 
 You can add the whole repository as an OpenWrt feed and add package to your firmware.
+
+Configuration
+-------------
+
+* **MAX_BROKERS** ``default: 10`` Maximum number of brokers that can be handled in a single process.
+
+    make CFLAGS="-D MAX_BROKERS=20"
+
+or you can edit the ``Makeflie`` and add
```

Comment:
There's no `Makefile` (at least none that anyone should edit by hand), the client is using cmake.

Also, I am not sure what the right way is to format this with reStructuredText, but the "or you can edit" part is not appropriately indented like the part before (you can [see it rendered here](https://github.com/wlanslovenija/tunneldigger/blob/2ef04245c913a741826eea4e6715265f2131c83e/docs/client.rst)).

Reply:
I mean the packages-opkg [``Makefile``](https://github.com/wlanslovenija/firmware-packages-opkg/tree/master/net/tunneldigger) 
I deleted the "or ..."